.github/workflows/testing.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
       with:
         files: .cache/unit/cover.txt,.cache/integration/cover.txt
         token: ${{ secrets.CODECOV_TOKEN }}
-        fail_ci_if_error: ${{ github.repository_owner == 'package-operator' }}
+        fail_ci_if_error: ${{ github.event_name != 'pull_request' }}
```

Comment:
shouldn't it be the opposite?
```yaml
fail_ci_if_error: ${{ github.event_name == 'pull_request' }}
```

Reply:
No. CI should NOT fail when this workflow is run during a pull request and this action error.

thus we want:
- for PR -> `fail_ci_if_error: false`
- for main branch tests and other things -> `fail_ci_if_error: true`

Reply:
my bad...I misread the title of the PR and thought it was the opposite